backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSGraphics.java

```diff
@@ -354,9 +355,18 @@ public void draw (GLKView view, CGRect rect) {
 		}
 
 		long time = System.nanoTime();
-		deltaTime = (time - lastFrameTime) / 1000000000.0f;
+		// After pause deltaTime can have somewhat huge value that destabilizes the mean, so let's cut it off

```

Comment:
Remove " that destabilizes the mean", this had sense on Android when we applied a mean on `delta`

Reply:
Good as done.